### This is the course website for Ecological and Evolutionary Genomics 2016
### Taught by Eric Collins (@rec3141), Assistant Professor of Biological Oceanography at the University of Alaska Fairbanks

### We are using GitHub as a repository for code developed in class and for background material related to the class.

### Bash shell and command line environment
- using BioLinux  
 - <https://github.com/Ecological-and-Evolutionary-Genomics/eeg2016/wiki/Jan-19-Intro-to-BioLinux>
- bash scripting  
 - <https://github.com/Ecological-and-Evolutionary-Genomics/eeg2016/wiki/Jan-23---Exercise-1>
  - <https://github.com/Ecological-and-Evolutionary-Genomics/eeg2016/wiki/Jan-25-Exercise-2>  

### Working in the cloud
- using Github for version control
 - <https://github.com/Ecological-and-Evolutionary-Genomics/eeg2016/wiki/Jan-23---Intro-to-Github>
- using BaseSpace for sequence analysis and storage
  - <https://github.com/Ecological-and-Evolutionary-Genomics/eeg2016/wiki/Jan-25---Into-to-BaseSpace>  
  - <https://github.com/Ecological-and-Evolutionary-Genomics/eeg2016/wiki/Feb-1---BaseSpace-Apps>  
- using IMG and MG-RAST for comparative genomics and metagenomics
  - <https://github.com/Ecological-and-Evolutionary-Genomics/eeg2016/wiki/April-6----Exercise-9-IMG---MG-RAST>  
- using Galaxy for comparative genomics and metagenomics
  - <https://github.com/Ecological-and-Evolutionary-Genomics/eeg2016/wiki/April-9----Exercise-10-Galaxy>  
- using the Arctic Region Supercomputing Center for taxonomic identification of short reads
  - <https://github.com/Ecological-and-Evolutionary-Genomics/eeg2016/wiki/ARSC-Tutorial>  

### Similarity searching
- theoretical background
 - <https://github.com/Ecological-and-Evolutionary-Genomics/eeg2016/wiki/Feb-22----similarity-searching>
- using BLAST and DIAMOND
  - <https://github.com/Ecological-and-Evolutionary-Genomics/eeg2016/wiki/Feb-22----Exercise-4-----BLAST-and-DIAMOND>

### Taxonomic identification  
- theoretical background
 - <https://github.com/Ecological-and-Evolutionary-Genomics/eeg2016/wiki/Feb-29----Taxonomic-identification-of-short-reads>
- from amplicon sequencing of ribosomal RNA genes, with mothur 
  - <https://github.com/Ecological-and-Evolutionary-Genomics/eeg2016/wiki/Feb-8---Exercise-3-mothur>
- from metagenomes, with k-mers using kraken
  - <https://github.com/Ecological-and-Evolutionary-Genomics/eeg2016/wiki/Feb-29----Exercise-5----kraken>
- from metagenomes, with conserved core genes using Phylosift 
  - <https://github.com/Ecological-and-Evolutionary-Genomics/eeg2016/wiki/Mar-7-Exercise-6----Phylosift>

### Genome sequencing
- genome assembly with SPAdes  
  - <https://github.com/Ecological-and-Evolutionary-Genomics/eeg2016/wiki/Mar-21-Exercise-7----SPAdes-assembler>  
- genome annotation with RAST  
  - <https://github.com/Ecological-and-Evolutionary-Genomics/eeg2016/wiki/March-28----Exercise-8-Genome-Annotation>  
